libpq/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ "$(uname)" == "Darwin" ]; then
+    export DYLD_LIBRARY_PATH=${PREFIX}/lib:${PREFIX}/lib64
+else
+    export LD_LIBRARY_PATH=${PREFIX}/lib:${PREFIX}/lib64
```

Comment:
We should ask about this in the conda mailing list.  Not sure how that will play out in the log run.